addons/point_of_sale/models/pos_order.py

```diff
@@ -938,6 +938,36 @@ def add_payment(self, data):
         self.amount_paid = sum(payment.amount for payment in self.statement_ids)
         return args.get('statement_id', False)
 
+    def _get_refund_date(self):
+        """
+        For potential inheritance.
+        """
+        return self._context.get('refund_date_order', fields.Datetime.now())
```

Comment:
while I understand the goal of the previous ones, we try to avoid one line hook methods, that makes the code more difficult to read. If somebody needs to modify this value, they can just override the `_prepare_refund_order_data` and change the returned value.

Reply:
I agree. I've just amended the commit to remove that one line hook method.